frontend/src/components/MapView/LeftPanel/layersPanel/MenuItem/MenuSwitch/SwitchItem/ExposureAnalysisOption.tsx

```diff
@@ -77,13 +86,15 @@ function ExposureAnalysisOption({
 
   return (
     <Tooltip title={t('Exposure Analysis') ?? ''}>
-      <IconButton
-        id={layer.id}
-        disabled={!selected}
-        onClick={handleExposureAnalysis}
-      >
-        <ImageAspectRatioOutlined />
-      </IconButton>
+      <>
```

Comment:
The changes in this file seem to break the tooltip (and the backend)

before:
<img width="260" alt="Screenshot 2023-12-05 at 6 25 36 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/1939a4b7-b2a5-4ca6-ac10-8e89f735d7f6">

after:
<img width="177" alt="Screenshot 2023-12-05 at 6 25 32 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/28c6c3cf-2726-408d-afd9-e078143ecc4e">

Reply:
I fixed this using actual spans